### PR TITLE
Describe new labels in triage guidelines

### PR DESCRIPTION
**Pull request by spzala**
Recently we have added new labels to identify issues that are candidates for close.
These issues are typically the one which are not goint to close via a fix. Add
description in triage guidelines about when to use these labels.
Related PR is,
https://github.com/kubernetes/test-infra/pull/7200

### CODE REVIEW

**Comment by cblecker** [comment]
/hold
We don't have commands for these new labels yet.

**Comment by spzala** [comment]
@cblecker thanks and I am good with hold but I thought we can use labels as such too i.e. use in combination with /close command. Org members can use these label followed by /close command to actually close the issue. Non-org members can not use close (or related) command but they can at least label trivial issues with their findings, and org member can close those issues quickly. 
Again, I am totally good to wait on it but this is how one of the usages I thought. Thanks!!

**Comment by cblecker** [comment]
@spzala The only way to apply these labels is to have write access to the repo. There is no prow command to add them.

We need to figure out the flow, and what we are trying to do. Are we trying to allow anyone to triage issues? Do we know if we have incentives for non-org members to triage issues? Or are these labels to track statistics on issues (how many duplicates, etc)

There's some confusion about the process, and we need to get it cleared up.

**Comment by spzala** [comment]
@cblecker thanks, and good questions. Based on PR, kubernetes/test-infra#7200, I thought these labels can be applied now. I was traveling lately so unfortunately didn't get to use them yet. So below: 
Are we trying to allow anyone to triage issues? 
> yes, we want to encourage anyone to look at open issues and triage. I am saying this based on the discussion when Jorge and I worked on creating triage guidelines.

Do we know if we have incentives for non-org members to triage issues? 
> I don't think that we have any incentives for non-org members and that's the one of the thoughts with these labels that it will help towards finding non-org members triage work.

Or are these labels to track statistics on issues (how many duplicates, etc)
> yes, exactly, that's one of the usages we had in mind and we explained in the initial google doc that we can display such statistics on devstat. 

Thanks!!

**Comment by jberkus** [comment]
Why is kind-support a kind, and not a close?

**Comment by cblecker** [comment]
Yeah, I would support a more broad "triage/*" series of labels.. like "triage/duplicate" or "triage/support-request"

File: contributors/guide/issue-triage.md [hunk 1]
```diff
@@ -39,10 +39,10 @@ and this document will cover the basic ones.
 ## Determine if it's a support request
 
 Sometimes users ask for support requests in issues; these are usually requests
-from people who need help configuring some aspect of Kubernetes. These should be
-directed to our support structures (see below) and then closed. Also, if the issue 
-is clearly abandoned or in the wrong place, it should be closed. Keep in mind that 
-only issue reporter, assignees and component organization members can close issue. 
+from people who need help configuring some aspect of Kubernetes. These issues should be
+labeled with `kind/support`, directed to our support structures (see below) and then closed. Also, if the issue
```
[the remark this answers is not in the document]
Comment (spzala) [reply contributors/guide/issue-triage.md]: Thanks, will fix it!

File: contributors/guide/issue-triage.md [hunk 2]
```diff
@@ -204,3 +206,18 @@ issues are work we would merge into the release if it gets done, but we wouldn't
...
+Issues that are identified as a support request, duplicate, not-reproducible or
+lacks enough information from reporter should be closed following guidelines explained
+in this file. Also, any issues that can not be resolved because of any particular reason
+should be closed. These issues should have one or more of following self-readable
+labels:
+* `kind/support`
+* `triage/duplicate`
+* `triage/not-reproducible`
+* `triage/needs-information`
+* `triage/unresolved`
```
[the remark this answers is not in the document]
Comment (spzala) [reply contributors/guide/issue-triage.md]: Sure I will add brief explanation here.

**Comment by cblecker** [comment]
/lgtm
/approve

@spzala Feel free to remove hold when you're ready for this to merge. I don't know if anyone else wants to review this as well.

**Comment by spzala** [comment]
@cblecker thanks!! The PR been open for review for some time now and we have discussed it in the ContribEx meeting so I am feeling safe to merge it. Also once it merge I can send email to ML about these changes we made from close/* to triage/* as we discussed earlier and can also use it at KubeCon if we end up having triage session at contributor summit.

**Pull request merged**